client/websocket_client.tsx

```diff
@@ -132,9 +150,39 @@ export default class WebSocketClient {
                     Reflect.deleteProperty(this.responseCallbacks, msg.seq_reply);
                 }
             } else if (this.eventCallback) {
-                if (msg.seq !== this.serverSequence && this.missedEventCallback) {
-                    console.log('missed websocket event, act_seq=' + msg.seq + ' exp_seq=' + this.serverSequence); //eslint-disable-line no-console
-                    this.missedEventCallback();
+                if (reliableWebSockets) {
+                    // We check the hello packet, which is always the first packet in a stream.
+                    if (msg.event === SocketEvents.HELLO && this.missedEventCallback) {
+                        console.log('got connection id ', msg.data.connection_id); //eslint-disable-line no-console
```

Comment:
There are 2 new `console.log`s here. They are deliberate and will help in debugging any issues for this feature. Hopefully, it should be fine.

Reply:
I'm fine with adding these. I think our "no console logs" rule might be a bit aggressive sometimes when they can be really helpful for debugging.